### PR TITLE
Test E2Es for Persian TV

### PR DESCRIPTION
**Pull request by LilyL0u**
**Overall change:**
Added brand and episode urls. Enabled on test.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
Passes on test

### CODE REVIEW

**Review by karinathomasbbc — APPROVED** [review]

File: cypress/support/config/settings.js [hunk 1]
```diff
@@ -4218,7 +4218,29 @@ module.exports = () => ({
...
+          test: {
+            paths: [
+              '/persian/bbc_persian_tv/tv_programmes/w13xttnr', // Brand
+              '/persian/bbc_persian_tv/tv/w172xbvktkvplq0', // Episode
+            ],
+            enabled: true,
+          },
+          local: {
+            paths: [],
+            enabled: true,
```
Comment (RichardPK) [inline cypress/support/config/settings.js]: What does local do if it's enabled but without any paths?

Comment (LilyL0u) [reply cypress/support/config/settings.js]: Doesn't run any tests, but I haven't run the full suite on local for a while to check how it works in that situation. I have disabled it for now and will come back to this in my local config audit

**Review by RichardPK — APPROVED** [review]

File: cypress/support/config/settings.js (hunk 1 above)
Comment (RichardPK) [reply cypress/support/config/settings.js]: Nice one!

Comment (karinathomasbbc) [reply cypress/support/config/settings.js]: If `enabled: true`, but path array is empty, it won't run any tests. So it doesn't really matter if it's disabled 😄

**Pull request merged**